x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -855,6 +881,12 @@ function generateNewAndRecoveredInstanceEvents<
   const recoveredAlertInstanceIds = Object.keys(recoveredAlertInstances);
   const newIds = without(currentAlertInstanceIds, ...originalAlertInstanceIds);
 
+  if (apm.currentTransaction) {
+    apm.currentTransaction.addLabels({
+      alerting_new_instances: newIds.length,
```

Comment:
Following our updated terminology, I believe this should be `alerting_new_alerts`, `alerting_active_alerts`, `alerting_recovered_alerts`